src/MDMPerforming.h

```diff
@@ -148,6 +148,30 @@ NS_SWIFT_NAME(ComposablePerforming)
 
 @end
 
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+/**
+ Provides the performer with a plan and an associated name.
+
+ @param plan The plan that required this type of performer.
+ @param name The name by which the plan can be identified.
+ */
+- (void)addPlan:(nonnull id<MDMPlan>)plan
+          named:(nonnull NSString *)name
+  NS_SWIFT_NAME(addPlan(plan:named:));
```

Comment:
Filed https://github.com/material-motion/material-motion-runtime-objc/issues/89 to track.